test/helpers/kubectl.go

```diff
@@ -420,9 +437,73 @@ func (kub *Kubectl) CiliumReport(namespace string, pod string, commands []string
 		fmt.Fprintln(wr, out.CombineOutput())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	kub.CiliumReportDump(namespace, pod)
 	return nil
 }
 
+//CiliumReportDump runs a variety of commands (CiliumCLICommands) and writes the results to
+//testResultsPath
+func (kub *Kubectl) CiliumReportDump(namespace string, pod string) {
+	reportEndpointCommands := map[string]string{
```

Comment:
I grok that what differentiates these commands from ciliumCLICommands is that they are output to console vs. the other commands are written to file. We can eliminate this and just use the ones that are written to file instead, as they include the output of `cilium endpoint get` and `cilium bpf policy list`.